src/app/shared/entity/cluster.ts

```diff
@@ -60,12 +60,25 @@ export class Cluster {
 
   static getVersionHeadline(type: string, isKubelet: boolean): string {
     if (type === 'kubernetes') {
-      return isKubelet ? 'kubelet Version' : 'Master Version';
+      return isKubelet ? 'kubelet Version' : 'Control Plane Version';
     }
 
     return '';
   }
 
+  // for now keep in sync with
+  // https://github.com/kubermatic/kubermatic/blob/master/pkg/webhook/cluster/validation/validation.go#L50-L52
```

Comment:
Will this change in future? Is there any API for that already?

Reply:
Yes, it should change in the future, but they definitely will not finish it within this release.
but also `for now, the sig-networking memebers will make sure to update the list of supported version both in the webhook and in the UI when adding a new CNI version.`